promotion/pom.xml

```diff
@@ -52,6 +52,11 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-test-autoconfigure</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Same as the above comment. I think we don't need this

Reply:
My mistake, I removed this dependency